recipe/activate.sh

```diff
@@ -7,33 +7,60 @@
 # doesn't come with a deactivate script, because the symlink
 # is benign and doesn't need to be deleted.
 
+_la_log() {
+    if [ "$CF_LIBARROW_ACTIVATE_LOGGING" = "1" ]; then
```

Comment:
Can you temporarily make this unconditional? I'd like to see a CI run through with the logging activated. Then we can revert that commit and merge.

Also, shouldn't this be `==`?